adapters/latest/build.gradle

```diff
@@ -18,10 +19,11 @@ test {
 }
 
 dependencies {
-    implementation(project(":common"))
+    api(project(":common"))
+    api("com.google.guava:guava:${project.rootProject.guavaVersion}")
     compileOnly('org.jetbrains:annotations:23.0.0')
     compileOnly('com.datastax.cassandra:cassandra-driver-core:3.11.3')
-    implementation("com.google.guava:guava:${project.rootProject.guavaVersion}")
+    implementation("org.apache.commons:commons-lang3:3.9")
```

Comment:
can we avoid adding this dependency? I generally get very nervous when we add new dependencies. So I would only add it if it's absolutely necessary.

Reply:
There were already a ton of usages of commons-lang3 in other subprojects so I don't think it's a big deal - the other ones are actually depending on some transitive dependency to pull it in it seems.